exports.go

```diff
@@ -20,13 +20,25 @@ import (
 	"time"
 )
 
+const (
+	// ResponseTypeSingleton is used for a service that sends a single response only
+	ResponseTypeSingleton = "Singleton"
+
+	// ResponseTypeStream is used for a service that will send multiple responses
+	ResponseTypeStream = "Stream"
+
+	// ResponseTypeChunked is used for a service that sends a single response in chunks (so not quite a stream)
+	ResponseTypeChunked = "Chunked"
+)
+
 // Export represents a single export
 type Export struct {
-	Name        string         `json:"name,omitempty"`
-	Subject     Subject        `json:"subject,omitempty"`
-	Type        ExportType     `json:"type,omitempty"`
-	TokenReq    bool           `json:"token_req,omitempty"`
-	Revocations RevocationList `json:"revocations,omitempty"`
+	Name         string         `json:"name,omitempty"`
+	Subject      Subject        `json:"subject,omitempty"`
+	Type         ExportType     `json:"type,omitempty"`
+	TokenReq     bool           `json:"token_req,omitempty"`
+	Revocations  RevocationList `json:"revocations,omitempty"`
+	ResponseType string         `json:"response_type,omitempty"`
```

Comment:
Should we make an enum? Would give us type checking in this lib at least.

Reply:
moved to responsetype - still using string because it is easier to write/read in the JSON